cmd/fsck.go

```diff
@@ -127,24 +132,32 @@ func fsck(ctx *cli.Context) error {
 		mpb.BarFillerClearOnComplete(),
 	)
 
-	keys := make(map[uint64]uint32)
 	var totalBytes uint64
-	for _, s := range slices {
-		bar.Increment()
-		keys[s.Chunkid] = s.Size
-		totalBytes += uint64(s.Size)
-		n := (s.Size - 1) / uint32(chunkConf.BlockSize)
-		for i := uint32(0); i <= n; i++ {
-			sz := chunkConf.BlockSize
-			if i == n {
-				sz = int(s.Size) - int(i)*chunkConf.BlockSize
-			}
-			key := fmt.Sprintf("%d_%d_%d", s.Chunkid, i, sz)
-			if _, ok := blocks[key]; !ok {
-				if _, err := blob.Head(key); err != nil {
-					logger.Errorf("can't find block %s: %s", key, err)
-					lost.Increment()
-					lostBytes.IncrBy(sz)
+	brokens := make(map[meta.Ino]string)
+	for inode, ss := range slices {
+		for _, s := range ss {
+			bar.Increment()
+			totalBytes += uint64(s.Size)
+			n := (s.Size - 1) / uint32(chunkConf.BlockSize)
+			for i := uint32(0); i <= n; i++ {
+				sz := chunkConf.BlockSize
+				if i == n {
+					sz = int(s.Size) - int(i)*chunkConf.BlockSize
+				}
+				key := fmt.Sprintf("%d_%d_%d", s.Chunkid, i, sz)
+				if _, ok := blocks[key]; !ok {
+					if _, err := blob.Head(key); err != nil {
+						if _, ok := brokens[inode]; !ok {
+							if p, st := m.GetPath(meta.Background, inode); st == 0 {
+								brokens[inode] = p
+							} else {
+								logger.Warnf("getpath of inode %d: %s", inode, st)
```

Comment:
use `inode:%d` as the path, otherwise it will be empty string

Reply:
It prints `$inode: $path` for each file, so maybe use the error message to replace path would be more comprehensive. Like:
```
juicefs[24127] <FATAL>: 3 objects are lost (24 bytes), 3 broken files:
        INODE: PATH
          325: /d3/f33
        32181: /fff
        32182: no such file or directory
```